components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -45,10 +45,10 @@ static CGFloat randomNumberNotEqualTo(const CGFloat targetNumber) {
 static UIColor *randomColor() {
   switch (arc4random_uniform(5)) {
     case 0:
-      return [UIColor whiteColor];
+      return [UIColor colorWithRed:1 green:1 blue:1 alpha:1];
       break;
     case 1:
-      return [UIColor blackColor];
+      return [UIColor colorWithRed:0 green:0 blue:0 alpha:1];
```

Comment:
What is the purpose of these two changes?

Reply:
The new unit test that verifies the layer's actual shadowColor uses `-colorWithCGColor:`, which places the color in the SRGB color space by default. `[UIColor blackColor]` and `-whiteColor` are in the Grey color space, and so object equality checks failed whenever those colors were used.